addons/stock/report/report_stock_forecasted.py

```diff
@@ -79,10 +79,9 @@ def _get_report_data(self, product_template_ids=False, product_variant_ids=False
         else:
             warehouse = self.env['stock.warehouse'].browse(self.get_warehouses()[0]['id'])
 
-        wh_location_ids = [loc['id'] for loc in self.env['stock.location'].search_read(
+        wh_location_ids = self.env['stock.location'].search(
```

Comment:
Open question:
`record_ids = search().ids` or `record_ids = list(_search())` (to avoid instantiating the recordset) ?

Reply:
imo the later is kinda an obscure micro-optimisation which isn't very readable, readability counts ! :D